stampifier.py

```diff
@@ -56,7 +56,8 @@ def __convert_website_to_stamp(self):
 
         LOGGER.debug(generated_stamp)
 
-        return generated_stamp
+        return generated_stamp, \
```

Comment:
It better to have a data model as `StampifierOutput`

Reply:
`StampifierResult`

Reply:
Updated! Thanks for suggestion